python/cmsis_svd/parser.py

```diff
@@ -305,7 +293,9 @@ def for_xml_file(cls, path):
         return cls(etree.parse(path))
 
     @classmethod
-    def for_packaged_svd(cls, package_root, vendor, filename):
+    def for_packaged_svd(
+        cls, package_root: str, vendor: str, filename: str
```

Comment:
`package_root` may be specified as a `PathLike` instead.

Reply:
Fixed.